### PR TITLE
update crds for spiderpool dra feature

### PR DESCRIPTION
**Pull request by cyclinder**
#### What type of PR is this?

- release/feature

**What this PR does / why we need it**:

update crds for spiderpool dra feature

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go [hunk 1]
```diff
@@ -95,6 +99,10 @@ type SpiderIPvlanCniConfig struct {
 	// +kubebuilder:validation:Optional
 	Bond *BondConfig `json:"bond,omitempty"`
 
+	// +kubebuilder:default=false
+	// +kubebuilder:validation:Optional
+	EnableRdma bool `json:"enableRdma"`
```
Comment (weizhoublue) [inline pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go:104]: I am wondering whether there should be a new field called "rdmaResourceName" here

Comment (cyclinder) [reply pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go]: done, maybe we can only use one field for this. If the value is empty, which means we don't use the rdma plugin. WDYT?

File: pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go [hunk 2]
```diff
@@ -79,6 +79,9 @@ type SpiderMacvlanCniConfig struct {
 	// +kubebuilder:validation:Optional
 	Bond *BondConfig `json:"bond,omitempty"`
 
+	// +kubebuilder:validation:Optional
```
Comment (weizhoublue) [inline pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go:82]: there should be another filed enableRdma

Comment (cyclinder) [reply pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go:82]: maybe we can only use one field for this. If the value is empty, which means we don't use the rdma plugin. WDYT?

Comment (weizhoublue) [reply pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go:82]: I do not think that way. Although, RdmaResourceName is a manual way. If no  enableRdma, the user has no way to tell whether turn on it or not , when it supports to automatically inject resource name

Comment (cyclinder) [reply pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go:82]: I'm ok with if we should keep consistent with `sriov`.

> If no enableRdma, the user has no way to tell whether turn on it or not , when it supports to automatically inject resource name

ah, I think the resourceName is not to be automatically injected, configures it only manually,  the Auto-injection means configuring the resourceName into the container's resources.

**Comment by cyclinder** [comment]
@weizhoublue ready to merge, please, this pr is pending https://github.com/spidernet-io/multus-cni/pull/2.

**Pull request merged**